axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/AxonServerEventStore.java

```diff
@@ -424,7 +424,9 @@ public TrackingEventStream openStream(TrackingToken trackingToken) {
                     .listEvents(new StreamObserver<EventWithToken>() {
                         @Override
                         public void onNext(EventWithToken eventWithToken) {
-                            logger.debug("Received event with token: {}", eventWithToken.getToken());
+                            logger.debug("Received event with token: {}. Stream Observer {}.",
```

Comment:
All other methods use `trace()`, this one `debug()`. Is that done on purpose?

Reply:
Nice catch! It was done like that already, but it makes more sense to be on a `trace` level.